### DISCUSSION TITLE
The effect of the length of (sub)database names on the speed of the library

### DISCUSSION THREAD

**Discussion by azarenkovgd** [kriszyp/lmdb-js#214] [Q&A] [answered]
Hi!

How do the names of (sub)databases (opened with db.openDb...) affect their performance (within a small margin of error, or seriously enough that it makes sense to name them as short as possible)?

Especially interested in the effect on read speed.

In my scenario, names can be up to ~100 characters long (several combined uuid).
I can try and drastically reduce the length - for example having map of long name/short identifier pairs (or something like that).
However, it would not be very convenient, and provided that the length of the name has almost no effect, I would prefer not to do it.

Thank you!

**Comment by kriszyp** [comment, marked as answer]
LMDB has pretty good performance with that size of key, so I would not expect 100 character database names to be a performance issue.

**Reply by azarenkovgd** [reply]
Thank you!
